scripts/events.js

```diff
@@ -57,12 +57,12 @@ $("#white-screen-restart-button")?.addEventListener("click", function () {
 
 document.addEventListener("keydown", () => {
   if (state.currentScreen === "preloader" || state.currentScreen === "whiteScreen") {
-    state.set({ currentScreen: "main" });
+    state.set({ currentScreen: "game", currentLocation: "main", currentMonsterIndex: null });
```

Comment:
so for the screen to show you have to temporarily disable the monsters index by setting it null?
wait i lowkey forgot what this eventlistener was targetting 😅